File: Freedom/我的简历/首页/ResumeDetailViewController.m
//  ResumeDetailViewController.m
//  Freedom
//  Created by Super on 16/9/5.
//  Copyright © 2016年 Super. All rights reserved.
//
#import "ResumeDetailViewController.h"
@interface ResumeDetailViewController ()<UIWebViewDelegate>{
    UIWebView *webView;
}
@end
@implementation ResumeDetailViewController
- (void)viewDidLoad {
    [super viewDidLoad];
    webView = [[UIWebView alloc]initWithFrame:CGRectMake(0, 0, APPW, APPH-TopHeight)];
    webView.delegate = self;
    webView.backgroundColor = [UIColor whiteColor];
    [webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:self.userInfo[@"url"]]]];
    [self.view addSubview:webView];
}
@end

File: Freedom/我的简历/首页/ResumeDetailViewController.swift
//
//  ResumeDetailViewController.swift
//  Freedom
//
//  Created by Super on 2018/5/15.
//  Copyright © 2018年 Super. All rights reserved.
//
import UIKit
import XExtension
class ResumeDetailViewController: ResumeBaseViewController,UIWebViewDelegate {
    let webView = UIWebView(frame: CGRect(x: 0, y: 0, width: APPW, height: APPH-TopHeight))
    override func viewDidLoad() {
        super.viewDidLoad()
        webView.delegate = self
        webView.backgroundColor = .white
        let request = URLRequest(url: URL(string: "http://u1610639.jisuapp.cn/app?_app_id=FPNRLS1lOl")!)
        webView.loadRequest(request)
        view.addSubview(webView)
    
    }
}
